Makefile

```diff
@@ -52,7 +52,7 @@ test-lint: # Lint files @Testing
 	poetry run ruff check manage_breast_screening
 
 test-ui: # Run UI tests @Testing
-	poetry run pytest -m system
+	poetry run pytest -m system --ignore manage_breast_screening/notifications/tests/dependencies
```

Comment:
I think we can ignore `manage_breast_screening/notifications`
We won't have system tests as these are UI related.